init.lua

```diff
@@ -50,6 +50,11 @@ mobs:register_mob("mobs_xenomorph:xenomorph", {
 		punch_start = 30,
 		punch_end = 59,
 	},
+	after_activate = function(self)
+		if not self.tamed then
+			self.type = "monster"
```

Comment:
kind of a hack :wink: but i think you tested it, right?

Reply:
Yes, tested. By `dump(self.type)` to-chat debug and by just gameplay

Reply:
I think the only side effect is that they will be able to spawn in peaceful mode, but that doesn't matter for pandorabox.